### PR TITLE
Create a release procedure

### LINKED ISSUE #63: Create a release procedure
Document the steps for releasing TS PNG on GitHub and on PyPi.

### PR DESCRIPTION
**Pull request by gabb222**
Fixes #63 

This procedure works if the developer has previously done a release.

### CODE REVIEW

**Comment by gabb222** [comment]
@Theia-Scientific/developers ready to review

File: RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# TS PNG Release Procedure
+
+Procedure for releasing a new TS PNG package on GitHub and on PyPi.
```
Comment (volks73) [inline RELEASE.md:3]: PyPi should be PyPI

File: RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+4. Commit this change with the message 'Add release date'.
+
+5. Change the package version in both `pyproject.toml` and `setup.py`.
+
+6. Commit these changes with the message 'Bump package version'.
+
+7. Publish the branch and create the pull request.
+
+8. When the PR is approved, merge the branch and locally pull the updated repository.
```
Comment (volks73) [inline RELEASE.md:21]: I would add "..., squash and merge the branch..."

File: RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+   ```sh
+   git push --tags
+   ```
+
+11. Go to the draft of the release on Github and publish it.
+
+12. Create the next milestone, move open issues to the next milestone, and close the previous milestone.
+
+## PyPi Release
```
Comment (volks73) [inline RELEASE.md:43]: PyPI

File: RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+3. Remove all files in the `dist` folder.
+
+4. Build the wheel.
+
+   ```sh
+   python3 -m build ./
+   ```
+
+5. Upload the build and enter PyPi credentials when prompted.
```
Comment (volks73) [inline RELEASE.md:65]: PyPI

**Review by volks73 — APPROVED** [review]
Looks good, some minor changes.

Approving so you can address and merge without waiting for me.

File: RELEASE.md (hunk 1 above)
Comment (gabb222) [reply RELEASE.md:3]: Change made as requested

File: RELEASE.md (hunk 3 above)
Comment (gabb222) [reply RELEASE.md:43]: Change made as requested

File: RELEASE.md (hunk 4 above)
Comment (gabb222) [reply RELEASE.md:65]: Change made as requested

**Pull request merged**